### PR TITLE
Add release notes 2021.02.24

### PR DESCRIPTION
**Pull request by bowang-666**

### CODE REVIEW

File: vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.3.0_v0.0.5
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.7.1
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.7.1-hotfix.20200403
+  - mcr.microsoft.com/k8s/flexvolume/keyvault-flexvolume:v0.0.13
+  - mcr.microsoft.com/k8s/flexvolume/blobfuse-flexvolume:1.0.15
+  - mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.2
+  - mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.3
+  - nginx:1.13.12-alpine
+  - mcr.microsoft.com/k8s/kms/keyvault:v0.0.9
+-rwxr-xr-x 1 root root 150582248 Jun  1  2020 /usr/local/bin/kubelet-1.17.3
```
Comment (robbiezhang) [inline vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt:111]: what's these?

Comment (bowang-666) [reply vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt:111]: https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh#L288
seems the logging is in mid. I'll send a PR to move it to a better position

**Review by adilliadil — COMMENTED** [review]
lgtm

File: vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  - mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.0
+  - mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.0.1
+  - mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:0.5.1
+  - mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:0.6.0
+  - mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:0.7.0
+  - mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.19
+  - mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12
+WARNING: 75% of /dev/sda1 is used
+Using kernel:
+Linux version 4.15.0-1106-azure (buildd@lgw01-amd64-057) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.12)) #118~16.04.1-Ubuntu SMP Tue Jan 19 16:13:06 UTC 2021
```
Comment (xuto2) [inline vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt:210]: FYI - kernel versions

**Pull request merged**